models/collection.js

```diff
@@ -5,8 +5,13 @@ const mutate = require('../lib/mutate');
 
 const BUILD_MODEL = require('./build').get;
 const PIPELINE_MODEL = require('./pipeline').get;
+const PRS = {
+    open: Joi.number().integer(),
```

Comment:
`.min(0)`?

Reply:
I was contemplating putting `.positive()` but wasn't sure if 0 would count or not so I just didn't add it 😂